pyproject.toml

```diff
@@ -18,12 +18,12 @@ exclude = '''
 
 [tool.isort]
 profile = "hug"
-src_paths = ["flake8_nb", "test"]
+src_paths = ["glotaran"]
```

Comment:
Copy pasta fix? Not sure why flake8_nb was involved before :)

Reply:
yeah 😅